README.md

```diff
@@ -1,70 +1,60 @@
-# Getting Started with Create React App
+# React Bookstore
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+> Use react to write JSX
+> Created Component for the todo list
+> Called TodoItem, Navbar, modules and TodoContainer to render out components
 
-## Available Scripts
+## Built With
 
-In the project directory, you can run:
+- React
 
-### `npm start`
+## Online live link
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+[Visit project online](https://github.com/mrparkersson/React-Bookstore)
```

Comment:
- [ ] the Live demo is not set up, please create a working link to the live demo or remove this part of the readme file entirely.